targets/frontend/src/components/layout/Navigation.tsx

```diff
@@ -23,86 +25,65 @@ export function Navigation() {
   const router = useRouter();
 
   const navigation: NavigationScheme = {
-    system: {
-      order: 1,
-      label: "Systèmes",
-      links: [{ href: "/users", label: "Gestion des utilisateurs" }],
-    },
     contents: {
-      order: 3,
+      order: 2,
       label: "Contenus",
       links: [
-        {
-          href: "/contenus",
-          label: "Contenus",
-        },
-        {
-          href: "/informations",
-          label: "Contenus éditoriaux",
-        },
-        {
-          href: "/models",
-          label: "Modèles de document",
-        },
-        {
-          href: "/agreements",
-          label: "Convention collectives",
-        },
-        {
-          href: "/contenus?source=highlights",
-          label: "À la une",
-        },
+        { href: "/contenus", label: "Tous les contenus" },
+        { href: "/contributions", label: "Contributions" },
```

Comment:
```suggestion
        { href: "/contributions", label: "Fiches pratiques" },
```

A voir avec le métier mais ça me parait plus logique